### ISSUE TITLE
Device Auth Url Link

### ISSUE THREAD

**Issue by RobSuperNimbus** [PlayEveryWare/eos_plugin_for_unity#25]
Hey, 

Im trying to figure out the device auth flow. I have it all working but when I call AuthInterface.Login() it prints out a URL to the console that I then manually have to copy and paste into my browser. Im trying to figure out either how to make this automatic so the browser window will popup for the user with the link supplied or where in the code this link gets printed so I can access it and pop up the browser myself in my own code. 

Any insight would be great. 

Regards
Rob

**Comment by adrianpew** [comment]
Hi Rob,

The EOS SDK should automatically pop up the browser in the current shipped version with the plugin (1.13.0).

Can you tell us more how you're logging in?  In the 'Auth & Friends' sample, we call into EOSManager.StartLoginWithLoginOptions (included in plugin):

https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/31ef5666da20314639e559af05674505f11c915b/Assets/Plugins/Source/EOSManager.cs#L671

**Comment by RobSuperNimbus** [comment]
Hey Adrian,

The Epic Online Services Version is 1.13.1 but the Epic Online Services Unity Plugin is 1.0.1. This correct? I changed my code to use what you have above EOSManager.Instance.StartLoginWithLoginOptions(loginOptions, (LoginCallbackInfo loginCallbackInfo) => {//some code}); but it still just prints out the url in the debug log and does not popup. Im running this in the Unity console. Its not a build or anything if that makes a difference. 

These are my login options: 
var loginOptions = new LoginOptions()
        {
            Credentials = new Credentials()
            {
                Type = LoginCredentialType.DeviceCode,
                Id = "RobSuperNimbus",
                Token = SystemInfo.deviceUniqueIdentifier.ToString()
            }
        };

**Comment by adrianpew** [comment]
Investigating which what behavior is expected from Epic API.

In the meantime, have you tried running the samples and see if Account Portal auth launches browser in your environment?

**Comment by RobSuperNimbus** [comment]
Yeah the account portal opens the browser. And yeah I meant the Unity Editor. But as I said the account portal opens from there so I assume the device auth should also.

**Comment by adrianpew** [comment]
https://dev.epicgames.com/docs/services/en-US/API/Members/Enums/Auth/EOS_ELoginCredentialType/index.html

"Deprecated and no longer used. Superseded by the EOS_LCT_ExternalAuth login method."

What platform are you targeting?

**Comment by RobSuperNimbus** [comment]
Im targeting windows at the minute. I would assume externalAuth is not a viable replacement for device auth as it seems to be for 3rd party authentication.
